holistic_bias/INSTRUCTIONS-count-biases.md

```diff
@@ -0,0 +1,52 @@
+# Multilingual Holistic Biases Counts
```

Comment:
Maybe you could add a link to https://github.com/facebookresearch/ResponsibleNLP/blob/main/holistic_bias/README.md to point to this document, so users can find it more easily?